app/views/admin/newsletters/index.html.erb

```diff
@@ -1,3 +1,43 @@
-<h2><%= t("admin.newsletters.index.title") %></h2>
+<h2 class="inline-block"><%= t("admin.newsletters.index.title") %></h2>
+<%= link_to t("admin.newsletters.index.new_newsletter"), new_admin_newsletter_path,
+                                                         class: "button float-right" %>
 
-<%= link_to t("admin.newsletters.index.button"), users_admin_newsletters_path, class: "button" %>
\ No newline at end of file
+<% if @newsletters.any? %>
```

Comment:
Every time I see an "If wadus exists" I expect an else that prints "There is no wadus" so the user knows what's going on in that scenario... do you think as well it would improve user experience?